CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ Note that this project **does not** adhere to [Semantic Versioning](https://semv
 
 ### Added
 
+- We added a new keyboard shortcut  <kbd>ctrl</kbd> + <kbd>,</kbd> to open the preferences. [#11154](https://github.com/JabRef/jabref/pull/11154)
+- [azatayamanaev](https://github.com/azatyamanaev) added value selection(like for month) for content selectors in custom entry types. [#11109](https://github.com/JabRef/jabref/issues/11109)
```

Comment:
We do not include our names here. Names are already part of the commit and committers will be highlighted at the release blog posts